lib/response_handler.rb

```diff
@@ -1,14 +1,16 @@
+require 'response'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.